src/components/contract/MemberContractAdminBlock.tsx

```diff
@@ -184,6 +183,7 @@ const useMemberContracts = (memberId: string) => {
           agreed_options
           revoked_at
           values
+          options
```

Comment:
NIT
I didn't see you use this parameter in your changes, why should you add it.